discord-scripts/fjord.ts

```diff
@@ -121,6 +123,58 @@ export default async function manageFjord(discordClient: Client, robot: Robot) {
   }
 
   if (process.env.HUBOT_N8N_WEBHOOK) {
+    if (discordClient.user) {
+      const handleGuild = async (guild: Guild) => {
+        await guild.members.fetch()
+
+        const membersList: {
+          username: string
+          nickname: string | null
+          id: string
+        }[] = []
+
+        guild.members.cache.forEach((member: GuildMember) => {
+          membersList.push({
+            username: member.user.username,
+            nickname: member.nickname,
+            id: member.id,
+          })
+        })
+
+        robot.logger.info(`Members of ${guild.name}:`)
+        robot.logger.info(JSON.stringify(membersList, null, 2))
```

Comment:
Let's not info-log potentially long data if we can avoid it. Debug level should be enough.